api/tools/src/main/java/org/pmiops/workbench/tools/ManageLeonardoRuntimes.java

```diff
@@ -53,18 +53,14 @@
   private static final Gson PRETTY_GSON = new GsonBuilder().setPrettyPrinting().create();
 
   private static Set<String> commaDelimitedStringToSet(String str) {
-    return Arrays.asList(str.split(",")).stream()
-        .filter(s -> !s.isEmpty())
-        .collect(Collectors.toSet());
+    return Arrays.stream(str.split(",")).filter(s -> !s.isEmpty()).collect(Collectors.toSet());
   }
 
   private static RuntimesApi newApiClient(String apiUrl) throws IOException {
     ApiClient apiClient = new ApiClient();
     apiClient.setBasePath(apiUrl);
-    GoogleCredential credential =
```

Comment:
This was the Deprecated library the warning referred to

Reply:
```
> Task :compile__tools__Java
Note: /Users/thibault/src/aou/api/tools/src/main/java/org/pmiops/workbench/tools/ManageLeonardoRuntimes.java uses or overrides a deprecated API.
```

Reply:
It's always a good idea to refresh the token, since it may have expired.

This new path still does: it includes a call to `credentials.refreshIfExpired()`